src/containers/Header/Header.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 
+import Sidebar from '../../components/Sidebar';
 import LocaleToggle from '../LocaleToggle';
 import ThemeSwitch from '../../components/ThemeSwitch';
 
 import './Header.css';
 
-const Header = () => (
-  <div className="header">
-    <LocaleToggle />
-    <ThemeSwitch />
-  </div>
-);
+class Header extends React.Component {
```

Comment:
Again, Sidebar should be on the same level as the Header, not within it as its part.

Reply:
moved above